index.html

```diff
@@ -0,0 +1,190 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+   
+   
+    <link rel="stylesheet" href="./css/styles.css" type="text/css" />
+    <link rel="stylesheet" href="./css/styleanim.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <title>African Cultural Dance Conference</title>
+</head>
+<body>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+    <div class="firefly"></div>
+   <!-- Header section start from here -->
+    <div class="contianer-fluid header-container">
+        <div class="container header-container1">
+             <header class="row d-flex justify-content-end">
+                 <div class="header-columns mypage"><a href="#" class="header-icons"><i class="fa fa-facebook"></i></a></div>
+                 <div class="header-columns mypage"><a href="#" class="header-icons"><i class="fa fa-twitter"></i></a></div>
+                 <div class="header-columns"><a href="#" class="header-text">English</a></div>
+                 <div class="header-columns"><a href="#" class="header-text">MyPage</a></div>
+                 <div class="header-columns"><a href="#" class="header-text">Logout</a></div>
+             </header>
+        </div>
+    </div>
+
+       <!-- navbar section start from here -->
+     <div class="contianer-fluid nav-bar-fluid-container">
+        <div class="container">
+             <nav class="row nav-bar-container">
+                <div class="col-2"> <img class="nav-logo" src="images/logo.svg" alt="Conference logo"></div>
+                <div class="col-10 d-flex justify-content-end"> 
+                    <ul class="nav-list">
+                        <li><a href="./about.html"> About</a></li>
+                        <li><a id="coursesiconindex" href="#main-courses-section"> Courses</a></li>
+                        <li><a id="joiniconindex" href="#"> Join</a></li>
+                        <li><a id="sponsericonindex" href="#"> Sponser</a></li>
+                        <li><a id="latesticonindex" href="#"> Latest</a></li>
+                        <li><a href="#"><button class="nav-bar-btn" type="submit">Register Now!</button></a></li>
+                    </ul>
+                </div>
+        
+             
+             </nav>
+        </div>
+     </div>
+
+     <!-- Main home page section start from here -->
+    <div class="contianer-fluid home-page-fluid-container" >
+        <div class="container-fluid menu-bar-fluid">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `style` your project using the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`. Your project must be the same as [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), you must make your project `color`, `shape`, `layout`, and `fonts` as shown on [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please correct them 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/235015529-b0f4abf1-288f-490f-93b0-e6d491bdc4c9.png)